drivers/bluetooth/hci/spi.c

```diff
@@ -304,8 +264,8 @@ static struct net_buf *bt_spi_rx_buf_construct(uint8_t *msg)
 			/* Event has not yet been handled */
 			__fallthrough;
 		default:
-			if (msg[1] == BT_HCI_EVT_LE_META_EVENT &&
-			    (msg[3] == BT_HCI_EVT_LE_ADVERTISING_REPORT)) {
+			if (msg[EVT_HEADER_EVENT] == BT_HCI_EVT_LE_META_EVENT &&
+			    (msg[EVT_VENDOR_CODE_LSB] == BT_HCI_EVT_LE_ADVERTISING_REPORT)) {
```

Comment:
This is not a vendor event code, so should not be using vendor event code defines

Reply:
Done.